libmscore/note.h

```diff
@@ -155,9 +155,20 @@ class NoteHead final : public Symbol
         ///\}
     };
 
+    enum class ParenthesesFlags : signed char {
+        ///.\{
```

Comment:
What's the meaning of having these comments?

Reply:
they are for generating the plugin documentation